call-in-sick/README.md

```diff
@@ -0,0 +1,120 @@
+# Call in sick with a custom bot for Hangouts Chat
+
+_Update your calendar and auto-responder with a single message to your custom
+bot._
+
+Last updated: August, 2019
+
+## Description
+
+When you wake up feeling ill the last thing you want to do is log in to your
+work account and update your status in multiple applications. With this
+solution, you simply send the message "I'm sick" to a custom bot in Hangouts
+Chat and it will turn on your out of office message in Gmail and block off
+your day in Google Calendar.
+
+<!-- TODO: Fix update image path to "master" before comitting. -->
+
+![demo](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@call-in-sick/call-in-sick/demo.png)
+
+## Technology highlights
+
+- Uses the [Hangouts Chat API][hangouts_chat_api] to build a chat bot, which you
+  can message just like a human participant.
+- The bot is implemented using [Apps Script][apps_script], allowing for rapid
+  development and worry-free operation.
+- Uses the [Gmail API][gmail_api] to set the user's out-of-office
+  auto-responder.
+- Blocks off the user's calendar using the [Calendar API][calendar_api].
+
+[hangouts_chat_api]: https://developers.google.com/hangouts/chat/
+[app_script]: https://developers.google.com/apps-script/
+[gmail_api]: https://developers.google.com/gmail/api/
+[calendar_api]: https://developers.google.com/calendar/
+
+## Try it
+
+First, create the script:
+
+1.  [Click here][code] to open the bot code in the Apps Script code editor.
+1.  Click the menu item **File > Make a copy** to get your own copy. You can
+    later access this copy in the [G Suite Developer Hub][hub].
+
+Then create a Google Cloud project to use with it:
+
+1.  [Create a new project][new_project] in the Google Cloud Developer Console.
+    Name it "Call in Sick", select a Billing Account if prompted, and
```

Comment:
Does Billing Account need to be bolded? 
https://developers.google.com/style/ui-elements

Reply:
Done